TrafficCapture/coreUtilities/src/main/java/org/opensearch/migrations/tracing/IWithStartTimeAndAttributes.java

```diff
@@ -0,0 +1,23 @@
+package org.opensearch.migrations.tracing;
+
+import io.opentelemetry.api.common.AttributesBuilder;
+import io.opentelemetry.api.metrics.DoubleHistogram;
+
+import java.time.Duration;
+import java.time.Instant;
+
+public interface IWithStartTimeAndAttributes extends IInstrumentationAttributes {
+    Instant getStartTime();
+
+    default Duration getSpanDuration() {
+        return Duration.between(getStartTime(), Instant.now());
```

Comment:
For duration metrics, I prefer using `System.nanoTime()` to better handle wall clock changes like during synchronization. Thoughts on storing a wall clock start time with instant as well as the System.nanoTime() start to use on durations?

Reply:
Thanks - good point

Reply:
Here's a nice history of nanoTime on parallel systems: https://stackoverflow.com/questions/510462/is-system-nanotime-completely-useless/54566928#54566928